docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -249,11 +253,11 @@ This configuration creates two users:
  * `duke` with a `dukePassword` password and a `reader` role
  * `john` with a `johnPassword` password and `reader` and `writer` roles
 
-`quarkus.oidc.client-id` and `quarkus.oidc.credentials.secret` can be used to customize the client id and secret.
+To customize the client id and secret, you can use the `quarkus.oidc.client-id` and `quarkus.oidc.credentials.secret` properties.
```

Comment:
IIRC, Sergey had asked me to keep the ID in client `id` lower case to help prevent confusion with the ID in "ID token".  Now, I've revisited this aspect of the document and believe it would be okay to capitalize ID consistently. I've updated the doc accordingly.